### PR TITLE
Add Azure Pipelines, Azure Boards, GitHub CodeQL Scanning status bardges

### PR DESCRIPTION
**Pull request by rudiansen**
Fixed [AB#340](https://dev.azure.com/rudiansen-lab/3710fb5f-9603-4b82-aeb1-38fc03e3101c/_workitems/edit/340)

### CODE REVIEW

**Comment by r-gunawan** [comment]
@rudiansen Is this already tested?

**Comment by rudiansen** [comment]
> 
> 
> @rudiansen Is this already tested?

Hi @r-gunawan, yeah, it's already tested and working fine.

**Comment by r-gunawan** [comment]
The change looks good.

**Review by r-gunawan — APPROVED** [review]
The change looks good.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ﻿# Parts Unlimited
 
+[![Build Status](https://dev.azure.com/rudiansen-lab/Parts%20Unlimited%20Demo/_apis/build/status/PartsUnlimited-CI?branchName=master)](https://dev.azure.com/rudiansen-lab/Parts%20Unlimited%20Demo/_build/latest?definitionId=9&branchName=master)
+[![Board Status](https://dev.azure.com/rudiansen-lab/3710fb5f-9603-4b82-aeb1-38fc03e3101c/e4881f68-9c92-4db5-8395-c9e986c9bb03/_apis/work/boardbadge/ac49a16e-e1e6-409f-8a99-0fb1a85e943c?columnOptions=1)](https://dev.azure.com/rudiansen-lab/3710fb5f-9603-4b82-aeb1-38fc03e3101c/_boards/board/t/e4881f68-9c92-4db5-8395-c9e986c9bb03/Microsoft.RequirementCategory/)
+
```
Comment (r-gunawan) [inline README.md:5]: @rudiansen Should we add a new line break here?

Comment (rudiansen) [reply README.md:5]: Yeah, just for eye looking.

**Pull request merged**